### PR TITLE
encapsulate element operation in HE page and improve the HE test cases

### PR DESCRIPTION
**Pull request by kenzhaoyihui**

### CODE REVIEW

File: page_objects/page_ovirt_hostedengine.py [hunk 1]
```diff
@@ -259,3 +262,211 @@ def check_additional_host_socre(self, ip, passwd):
...
+
+            # FINISH STAGE
+            self.click(self.FINISH_DEPLOYMENT)
+            self.click(self.CLOSE_BUTTON, 1500)
+
+        prepare_env()
+        check_deploy()
+
+    def check_hostedengine_deployed(self):
+        self.assert_element_visible(self.HE_RUNNING)
```
Comment (qiyuann) [inline page_objects/page_ovirt_hostedengine.py]: assertions should be moved to test

File: page_objects/page_ovirt_hostedengine.py [hunk 2]
```diff
@@ -259,3 +262,211 @@ def check_additional_host_socre(self, ip, passwd):
...
+        self.input_text(self.ADMIN_PASS, self.config_dict['admin_pass'])
+        self.click(self.NEXT_BUTTON)
+
+        # PREPARE VM
+        self.click(self.PREPARE_VM_BUTTON)
+        self.click(self.NEXT_BUTTON, 1500)
+
+    def node_zero_default_deploy_process(self):
+        def prepare_env():
+            self.move_failed_setup_log()
```
Comment (qiyuann) [inline page_objects/page_ovirt_hostedengine.py]: I suggest to encapsulate only one prepare_env method for all processes, this method should deal with cleaning of different storages

File: page_objects/page_ovirt_hostedengine.py (hunk 1 above)
Comment (kenzhaoyihui) [reply page_objects/page_ovirt_hostedengine.py]: solved.

File: page_objects/page_ovirt_hostedengine.py (hunk 2 above)
Comment (kenzhaoyihui) [reply page_objects/page_ovirt_hostedengine.py]: solved

**Pull request merged**